### ISSUE TITLE
The Pending tab uses the Canvas course ID instead of the course name in the ARIA label

### ISSUE THREAD

**Issue by Rainribbon** [Kaltura-EDU/issues-tracking#15]
Submitted in Kaltura Support Case #00548840:

When using a screen reader in the Media Gallery, the tabs are described as "4 Media in Media Gallery <course ID>" or "Playlists in Media Gallery <course ID>". Screen reader users may find the course ID in this label confusing, as the ID is a string of numbers that may be confused for the number of media items. For example, "Playlists in Media Gallery 24" may sound like there are 24 items in the Media Gallery, when this actually the course ID. For greater clarity, the course ID should be replaced with the course name in the ARIA label. For example, "Playlists in Media Gallery 22Su History of UVA".

**Comment by Rainribbon** [comment]
This issue is being tracked by Kaltura in ADA-2822.
